feat: Add `error_message`, `provider_account_id`, and `interface_endpoint_name` to Stream PrivateLinkEndpoint models (#3161)
mongodb/terraform-provider-mongodbatlas: docs/data-sources/stream_privatelink_endpoint.md

- `dns_domain` (String) Domain name of Privatelink connected cluster.
- `dns_sub_domain` (List of String) Sub-Domain name of Confluent cluster. These are typically your availability zones.
- `error_message` (String) Error message if the connection is in a failed state.
- `interface_endpoint_id` (String) Interface endpoint ID that is created from the specified service endpoint ID.

> - `interface_endpoint_name` (String) Interface endpoint name that is created from the specified service endpoint ID.

Reviewer: ```suggestion
- `interface_endpoint_name` (String) Name of interface endpoint created from the specified service endpoint ID.
```

Reviewer 2: this should now be changed, are there any other questions related to docs @lmkerbey-mdb?
